lib/refills/import_generator.rb

```diff
@@ -21,8 +22,14 @@ def copy_styles
     end
 
     def copy_javascripts
+      source =
+        if coffee?
+          File.join("javascripts", "refills", "coffeescript", javascript_name)
+        else
+          File.join('javascripts', 'refills', javascript_name)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.